opencti-platform/opencti-front/src/private/components/analyses/notes/NotePopover.tsx

```diff
@@ -129,13 +151,21 @@ const NotePopover: FunctionComponent<NotePopoverProps> = ({
               {t_i18n('Remove from this entity')}
             </MenuItem>
           )}
+          <Security needs={[KNOWLEDGE_KNENRICHMENT]}>
+            <MenuItem onClick={handleOpenEnrichment}>
+              {t_i18n('Enrich')}
+            </MenuItem>
+          </Security>
+          <MenuItem onClick={handleOpenEnroll}>{t_i18n('Enroll in playbook')}{!isEnterpriseEdition && <EEChip />}</MenuItem>
```

Comment:
EEChip already return null if isEntrepriseEdition.

Also, if we add this here we need ti in every popover on all entities.

AS we wil move rapidly to quickactions I suggest we remove the chip entirely to avoid some cumbersome work that will not stay long.